src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -409,6 +447,27 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 			if (outputElement) {
 				outputElement.focus();
 			}
+			// Move cursor to the last known location
```

Comment:
There's a lot of logic here and it's not immediately obvious to me what this is supposed to be doing. Could you please add a bit of explanation to make this clearer?

Reply:
Sure thing, I'll add comments to make it decipherable! (I didn't find an easier way to get and set the cursor location in a content editable div)